### DISCUSSION TITLE
how do i set a child (q-splitter or div) to take the full available height

### DISCUSSION THREAD

**Discussion by knightsrule** [quasarframework/quasar#17646] [General - Components / Directives / etc]
here is a simple layout that I am trying. the q-page shows up grey background using the full height and width available. The however, the child div only takes up the height needed to display the text. 

	<q-page class="column bg-grey">
		<div class="full-height bg-green col-auto">test</div>

This is a simple test. I actually need a q-splitter to show the same way, but div is easy to test out. 

Thanks in advance!

**Comment by hubdidub** [comment]
Try removing `full-height` and also switch out `col-auto` with `col`.
The job of `col-auto` is to make the cell fill only the space it needs to be rendered. That contradicts the `full-height` class.
